practice-two/src/index.html

```diff
@@ -320,6 +320,85 @@ <h3 class="project__item-title">Watch apps</h3>
       <section></section>
       <!-- END REVIEW SECTION-->
 
+    <!-- START REVIEW SECTION -->
+    
+    <section class="review">
+      <div class="container review__wrapper">
+        <h2 class="heading-section text-color-secondary">Testimonials</h2>
+        <div class="review__content">
+          <div class="text-color-secondary review__card review__card--light">
+            <div class="review__card-description">
+              <p class="text-normal people-comment">
+                It is not because things are difficult that we do not dare; it is because we do not dare that they are
+                difficult.
+              </p>
+              <p class="review__card-name">
+                Seneca the Younger
+              </p>
+              <p class="text-normal">
+                Editor in okodesign.net
+              </p>
+            </div>
+            <figure class="review__person-img">
+              <img src="assets/images/person-review-light.webp" alt="person review card light"
+                class="img-default img-border">
+            </figure>
+          </div>
+          <div class="review__card review__card--platinum">
+            <div class="review__card-description">
+              <p class="text-color-base text-normal people-comment">
+                Attending a trade show can be a very effective method of promoting your company and its products.
+              </p>
+              <p class="text-color-primary review__card-name card_name-platinum ">
+                Rachel Moreno
+              </p>
+              <p class="text-color-base text-normal">
+                Lead Designer in okodesign.net
+              </p>
+            </div>
+            <figure class="review__person-img">
+              <img src="assets/images/person-review-paltinum.webp" alt="person review card platium"
+               class="img-default img-border">
+            </figure>
+          </div>
+        </div>
+          <ul class="review__icon">
+            <li>
+              <svg class="dotnav-icon">
+                <use xlink:href="assets/images/icon-review.svg#dotnav"></use>
+               </svg>
+            </li>
+            <li>
+              <svg class="dotnav-icon">
+                <use xlink:href="assets/images/icon-review.svg#dotnav"></use>
+              </svg>
+            </li>
+            <li>
+              <svg class="dotnav-icon">
+                <use xlink:href="assets/images/icon-review.svg#dotnav"></use>
+              </svg>
+            </li>
+            <li>
+              <svg class="dotnav-icon icon-hidden">
```

Comment:
You should use BEM naming convention for icon-hidden. Eg `dot-icon dot-icon--hidden`